common-lib/src/cmr/common/system.clj

```diff
@@ -46,19 +46,19 @@
   starting and started"
   [system-name component-order]
   (fn [s]
-    (info (str system-name " System starting"))
+    (reportf "%s System starting" system-name)
```

Comment:
Are you changing the level categorization from INFO to REPORT here? If so, why?

Reply:
I'm condensing two logs into one, these fire for the start and stop events and look redundant with the others. As for why the level, I think it is important to always know if a server is going up and down. I don't think this adds that much to logs as it happens just once. I also think it is needed to debug which sever has failed to come up locally. (we run in info on AWS from what I understand, so no change to production, only locally where I think it is important to see that the boot process has failed to start properly).

Reply:
one problem I have with a bunch of these logs is that there is no way of connecting which search node is going down. I added :instance-name to the system object and would like to look into using that to be able to better report which node is shutting down, or more important, being able to count exactly how many match start ups and shutdowns there are so we can tell how AWS killed the software (forcibly or gracefully).

Reply:
OK. Currently the task IP address is emitted with the log message. Any improvement you add to help identify the task is fine by me.
2024-01-23T21:56:11.949Z ip-10-6-65-205.ec2.internal INFO [cmr.search.system:?] - search System starting